[2022-06-11 10:59:16,549] [builder.py:185]            [INFO]     |====================================================================================================|
[2022-06-11 10:59:16,550] [builder.py:185]            [INFO]     |                                    最佳華語電影一百部 Collection in 亚洲电影                                    |
[2022-06-11 10:59:16,551] [builder.py:185]            [INFO]     |====================================================================================================|
[2022-06-11 10:59:16,552] [builder.py:186]            [INFO]     |                                                                                                    |
[2022-06-11 10:59:16,553] [builder.py:191]            [INFO]     |================================= Validating 最佳華語電影一百部 Attributes ==================================|
[2022-06-11 10:59:16,554] [builder.py:437]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:16,554] [builder.py:438]            [DEBUG]    | Validating Method: sync_mode                                                                       |
[2022-06-11 10:59:16,555] [builder.py:442]            [DEBUG]    | Value: sync                                                                                        |
[2022-06-11 10:59:16,556] [builder.py:582]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:16,556] [builder.py:583]            [DEBUG]    | Validating Method: collection_order                                                                |
[2022-06-11 10:59:16,556] [builder.py:584]            [DEBUG]    | Value: custom                                                                                      |
[2022-06-11 10:59:16,558] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:16,559] [builder.py:604]            [DEBUG]    | Validating Method: trakt_list                                                                      |
[2022-06-11 10:59:16,560] [builder.py:605]            [DEBUG]    | Value: https://trakt.tv/users/bpsim/lists/hkfa-s-the-best-100-chinese-motion-pictures              |
[2022-06-11 10:59:17,604] [builder.py:600]            [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:17,605] [builder.py:604]            [DEBUG]    | Validating Method: url_poster                                                                      |
[2022-06-11 10:59:17,605] [builder.py:605]            [DEBUG]    | Value: https://static.careerengine.us/api/aov2/https%3A_%7C__%7C_mmbiz.qpic.cn_%7C_mmbiz_gif_%7C_kpy7ZuXkoOuFTDI2ZBriaoxXmfS3s8nvoL53ClSG8DLZXCP7RxmBEwpOwSzQRmicu3TZtHn3cicXibkuENpkwibicXqQ_%7C_640%3Fwx_fmt%3Dgif.lg.jpeg |
[2022-06-11 10:59:17,795] [builder.py:791]            [WARNING]  | Collection Warning: No Poster Found at https://static.careerengine.us/api/aov2/https%3A_%7C__%7C_mmbiz.qpic.cn_%7C_mmbiz_gif_%7C_kpy7ZuXkoOuFTDI2ZBriaoxXmfS3s8nvoL53ClSG8DLZXCP7RxmBEwpOwSzQRmicu3TZtHn3cicXibkuENpkwibicXqQ_%7C_640%3Fwx_fmt%3Dgif.lg.jpeg |
[2022-06-11 10:59:20,092] [plex.py:742]               [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:20,093] [plex.py:745]               [DEBUG]    | Looking for: 最佳華語電影一百部                                                                             |
[2022-06-11 10:59:20,093] [builder.py:764]            [INFO]     |                                                                                                    |
[2022-06-11 10:59:20,094] [builder.py:765]            [INFO]     | Validation Successful                                                                              |
[2022-06-11 10:59:20,094] [plex_meta_manager.py:539]  [INFO]     |                                                                                                    |
[2022-06-11 10:59:20,095] [plex_meta_manager.py:541]  [INFO]     |=================================== Running 最佳華語電影一百部 Collection ===================================|
[2022-06-11 10:59:20,095] [plex_meta_manager.py:553]  [INFO]     |                                                                                                    |
[2022-06-11 10:59:20,096] [plex_meta_manager.py:554]  [INFO]     | Sync Mode: sync                                                                                    |
[2022-06-11 10:59:20,096] [plex_meta_manager.py:557]  [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:20,097] [plex_meta_manager.py:558]  [DEBUG]    | Builder: trakt_list: https://trakt.tv/users/bpsim/lists/hkfa-s-the-best-100-chinese-motion-pictures |
[2022-06-11 10:59:20,097] [plex_meta_manager.py:559]  [INFO]     |                                                                                                    |
[2022-06-11 10:59:20,097] [trakt.py:482]              [INFO]     | Processing Trakt List: https://trakt.tv/users/bpsim/lists/hkfa-s-the-best-100-chinese-motion-pictures |
[2022-06-11 10:59:20,392] [builder.py:1507]           [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:20,393] [builder.py:1508]           [DEBUG]    | 104 IDs Found: [(21058, 'tmdb'), (11471, 'tmdb'), (18311, 'tmdb'), (92990, 'tmdb'), (49982, 'tmdb'), (69599, 'tmdb'), (104237, 'tmdb'), (72445, 'tmdb'), (44154, 'tmdb'), (146, 'tmdb'), (112699, 'tmdb'), (15804, 'tmdb'), (74100, 'tmdb'), (45871, 'tmdb'), (54178, 'tmdb'), (11713, 'tmdb'), (161285, 'tmdb'), (176212, 'tmdb'), (13345, 'tmdb'), (21835, 'tmdb'), (157243, 'tmdb'), (68883, 'tmdb'), (11104, 'tmdb'), (176217, 'tmdb'), (45999, 'tmdb'), (42006, 'tmdb'), (176213, 'tmdb'), (140260, 'tmdb'), (37185, 'tmdb'), (112698, 'tmdb'), (151243, 'tmdb'), (176219, 'tmdb'), (10775, 'tmdb'), (11230, 'tmdb'), (44914, 'tmdb'), (40751, 'tmdb'), (26005, 'tmdb'), (176220, 'tmdb'), (107248, 'tmdb'), (157252, 'tmdb'), (38155, 'tmdb'), (78450, 'tmdb'), (10835, 'tmdb'), (10617, 'tmdb'), (68004, 'tmdb'), (38143, 'tmdb'), (176228, 'tmdb'), (176230, 'tmdb'), (176231, 'tmdb'), (64015, 'tmdb'), (30421, 'tmdb'), (176313, 'tmdb'), (176317, 'tmdb'), (154633, 'tmdb'), (42120, 'tmdb'), (10997, 'tmdb'), (25538, 'tmdb'), (176323, 'tmdb'), (176325, 'tmdb'), (76294, 'tmdb'), (9056, 'tmdb'), (107682, 'tmdb'), (9261, 'tmdb'), (80427, 'tmdb'), (157244, 'tmdb'), (176617, 'tmdb'), (176626, 'tmdb'), (104287, 'tmdb'), (157236, 'tmdb'), (81593, 'tmdb'), (176628, 'tmdb'), (141906, 'tmdb'), (176629, 'tmdb'), (157229, 'tmdb'), (45996, 'tmdb'), (118212, 'tmdb'), (11770, 'tmdb'), (30992, 'tmdb'), (176630, 'tmdb'), (104681, 'tmdb'), (106380, 'tmdb'), (44162, 'tmdb'), (176631, 'tmdb'), (157227, 'tmdb'), (176632, 'tmdb'), (176633, 'tmdb'), (122973, 'tmdb'), (16269, 'tmdb'), (24163, 'tmdb'), (18329, 'tmdb'), (843, 'tmdb'), (95372, 'tmdb'), (176638, 'tmdb'), (176640, 'tmdb'), (170931, 'tmdb'), (176642, 'tmdb'), (152421, 'tmdb'), (143479, 'tmdb'), (32520, 'tmdb'), (176643, 'tmdb'), (57564, 'tmdb'), (96712, 'tmdb'), (72507, 'tmdb'), (36210, 'tmdb')] |
[2022-06-11 10:59:20,394] [builder.py:1509]           [DEBUG]    |                                                                                                    |
[2022-06-11 10:59:20,438] [builder.py:2009]           [INFO]     |                                                                                                    |
[2022-06-11 10:59:20,442] [builder.py:2010]           [INFO]     |================================== Adding to 最佳華語電影一百部 Collection ==================================|
[2022-06-11 10:59:20,443] [builder.py:2011]           [INFO]     |                                                                                                    |
[2022-06-11 10:59:20,443] [builder.py:2025]           [INFO]     |  1/39 | 最佳華語電影一百部 Collection | + | 小城之春 (1948)                                                     |
[2022-06-11 11:00:14,504] [builder.py:2025]           [INFO]     |  2/39 | 最佳華語電影一百部 Collection | + | 黄土地 (1984)                                                      |
[2022-06-11 11:01:14,637] [plex_meta_manager.py:668]  [DEBUG]    | Traceback (most recent call last):                                                                 |
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 449, in _make_request
                                                                 |     six.raise_from(e, None)
                                                                 |   File "<string>", line 3, in raise_from
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 444, in _make_request
                                                                 |     httplib_response = conn.getresponse()
                                                                 |   File "/usr/lib/python3.8/http/client.py", line 1386, in getresponse
                                                                 |     response.begin()
                                                                 |   File "/usr/lib/python3.8/http/client.py", line 312, in begin
                                                                 |     version, status, reason = self._read_status()
                                                                 |   File "/usr/lib/python3.8/http/client.py", line 273, in _read_status
                                                                 |     line = str(self.fp.readline(_MAXLINE + 1), "iso-8859-1")
                                                                 |   File "/usr/lib/python3.8/socket.py", line 669, in readinto
                                                                 |     return self._sock.recv_into(b)
                                                                 | socket.timeout: timed out
                                                                 | 
                                                                 | During handling of the above exception, another exception occurred:
                                                                 | 
                                                                 | Traceback (most recent call last):
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 440, in send
                                                                 |     resp = conn.urlopen(
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 785, in urlopen
                                                                 |     retries = retries.increment(
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/util/retry.py", line 550, in increment
                                                                 |     raise six.reraise(type(error), error, _stacktrace)
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/packages/six.py", line 770, in reraise
                                                                 |     raise value
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 703, in urlopen
                                                                 |     httplib_response = self._make_request(
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 451, in _make_request
                                                                 |     self._raise_timeout(err=e, url=url, timeout_value=read_timeout)
                                                                 |   File "/usr/lib/python3.8/site-packages/urllib3/connectionpool.py", line 340, in _raise_timeout
                                                                 |     raise ReadTimeoutError(
                                                                 | urllib3.exceptions.ReadTimeoutError: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
                                                                 | 
                                                                 | During handling of the above exception, another exception occurred:
                                                                 | 
                                                                 | Traceback (most recent call last):
                                                                 |   File "plex_meta_manager.py", line 570, in run_collection
                                                                 |     items_added, items_unchanged = builder.add_to_collection()
                                                                 |   File "/root/Plex-Meta-Manager/modules/builder.py", line 2032, in add_to_collection
                                                                 |     self.library.alter_collection(item, name, smart_label_collection=self.smart_label_collection)
                                                                 |   File "/root/Plex-Meta-Manager/modules/plex.py", line 650, in alter_collection
                                                                 |     field = next((f for f in item.fields if f.name == "collection"), None)
                                                                 |   File "/usr/lib/python3.8/site-packages/plexapi/base.py", line 483, in __getattribute__
                                                                 |     self._reload(_overwriteNone=False)
                                                                 |   File "/usr/lib/python3.8/site-packages/plexapi/base.py", line 357, in _reload
                                                                 |     data = self._server.query(key)
                                                                 |   File "/usr/lib/python3.8/site-packages/plexapi/server.py", line 696, in query
                                                                 |     response = method(url, headers=headers, timeout=timeout, **kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 542, in get
                                                                 |     return self.request('GET', url, **kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 529, in request
                                                                 |     resp = self.send(prep, **send_kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/sessions.py", line 645, in send
                                                                 |     r = adapter.send(request, **kwargs)
                                                                 |   File "/usr/lib/python3.8/site-packages/requests/adapters.py", line 532, in send
                                                                 |     raise ReadTimeout(e, request=request)
                                                                 | requests.exceptions.ReadTimeout: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
                                                                 | 
[2022-06-11 11:01:14,699] [plex_meta_manager.py:669]  [ERROR]    | Unknown Error: HTTPConnectionPool(redacted): Read timed out. (read timeout=60)
[2022-06-11 11:01:14,700] [plex_meta_manager.py:674]  [INFO]     | 
[2022-06-11 11:01:14,701] [plex_meta_manager.py:675]  [INFO]     |====================================================================================================|
[2022-06-11 11:01:14,702] [plex_meta_manager.py:675]  [INFO]     |                                   Finished 最佳華語電影一百部 Collection                                    |
[2022-06-11 11:01:14,703] [plex_meta_manager.py:675]  [INFO]     |                                    Collection Run Time: 0:01:58                                    |
[2022-06-11 11:01:14,704] [plex_meta_manager.py:675]  [INFO]     |====================================================================================================|
